ragondin/routers/openai.py

```diff
@@ -74,13 +74,15 @@ async def list_models(
 def __get_partition_name(model_name, app_state):
     if not model_name.startswith("ragondin-"):
         raise HTTPException(
-            status_code=status.HTTP_404_NOT_FOUND, detail="Model not found"
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail="Model not found. Model should respect this format `ragondin-{partition}`",
```

Comment:
This is unrelated to this PR scope, but I just noticed that when I give a wrong model, I receive `{"detail":"Error: 404: Model not found"}`, but a `500 Internal Server Error` in the HTTP header